linkingFuncs.py

```diff
@@ -149,9 +186,8 @@ def remove_watchlist(username, tik):
     if (id ==  None):
         return { 'Action': False}   
     time = datetime.datetime.now()
-    cur.execute("INSERT INTO Transactions Values ('%d', '%s', 0, 'SELL, '%f', '%s', 0)" % (
-        id, tik, price, time))
-    cur.execute("UPDATE * FROM Transactions SET display=0 WHERE ticker='%s' AND amount=0 AND display=1" % (tik))
+    cur.execute("INSERT INTO Transactions Values ('%d', '%s', 4, 'SELL', '%f', '%s', 0)" % (id, tik, price, time))
```

Comment:
Out of curiosity, are the 4 and 0 hard coding a value or what do they represent?

Reply:
LGTM

Reply:
Made the change for the next push